File: CursoCsharpUdemy/exer/udemy/EstruturaCondicional/Exer02.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace exer.udemy.EstruturaCondicional
{
    public class Exer02
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Informe um número: ");
            int number = int.Parse(Console.ReadLine());

            if (number % 2 == 0)
            {
                Console.WriteLine("PAR");
            }
            else
            {
                Console.WriteLine("IMPAR");
            }
        }
    }
}

File: CursoJava/Repeticao-Sequencial-Condicional/src/Exer/EstruturaCondicional/Exer02.java
package Exer.EstruturaCondicional;

import java.util.Scanner;

public class Exer02 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		System.out.println("Digite um número inteiro: ");
		int number = sc.nextInt();
		
		if(number % 2 == 0) {
			System.out.println(number + " é par.");
		} else {
			System.out.println(number + " é impar.");
		}		
		
		sc.close();
	}

}
